test/parse.js

```diff
@@ -19,7 +19,12 @@ var Path = require('path');
 // http://mxr.mozilla.org/mozilla-central/source/js/src/tests/ecma_5/JSON/
 
 var dirsPath = Path.resolve(__dirname, 'parse-cases');
-var dirs = FS.readdirSync(dirsPath);
+var dirs = FS.readdirSync(dirsPath).filter(function(fn) {
+  if (fn.startsWith('.')) {
```

Comment:
This is why Travis CI is failing for Node 0.10 and 0.12. String.prototype.startsWith wasn't added until ES2015.